dashboard/client/src/pages/serve/ServeApplicationRow.tsx

```diff
@@ -56,7 +56,7 @@ export const ServeApplicationRow = ({
       <TableCell align="center">
         <CodeDialogButton
           title={name ? `Application config for ${name}` : `Application config`}
-          code={deployed_app_config}
+          code={deployed_app_config || "null"}
```

Comment:
can we update this also to return `-` if there is no app config.

Reply:
<img width="1711" alt="image" src="https://user-images.githubusercontent.com/125417081/236898295-5b5595da-9ed1-4e35-8dc7-5affd660a25f.png">
Done